documentation/asciidoc/accessories/ai-camera/details.adoc

```diff
@@ -15,7 +15,7 @@ To fully understand this system, familiarise yourself with the following concept
 
 Input Tensor:: The part of the sensor image passed to the AI engine for inferencing. Produced by a small on-board ISP which also crops and scales the camera image to the dimensions expected by the neural network that has been loaded. The input tensor is not normally made available to applications, though it is possible to access it for debugging purposes.
 
-Region of Interest (ROI):: Specifies exactly which part of the sensor image is cropped out before being rescaled to the size demanded by the neural network. Can be queried and set by an application. The units used are always pixels in the full resolution sensor output. The default ROI setting uses the full image received from the sensor, cropping no data.
+Region of Interest (ROI):: Specifies exactly which part of the sensor image is cropped out before being rescaled to the size demanded by the neural network. Can be queried and set by an application. The units used are always pixels in the full resolution sensor output. The default region of interest (ROI) setting uses the full image received from the sensor, cropping no data.
```

Comment:
I don't think we need to explain the ROI acronym again _within_ the definition of ROI (which is already expanded on this very same line).